src/main/resources/migrations/00012_add_fields_to_transaction_table.sql

```diff
@@ -0,0 +1,12 @@
+--liquibase formatted sql
+
+--changeset uk.gov.pay:add_fields_to_transaction_table
+
+ALTER TABLE transaction
+    ADD COLUMN net_amount BIGINT,
+    ADD COLUMN total_amount BIGINT,
+    ADD COLUMN settlement_submitted_time TIMESTAMP WITH TIME ZONE,
+    ADD COLUMN settled_time TIMESTAMP WITH TIME ZONE,
+    ADD COLUMN refund_status TEXT,
```

Comment:
(not a blocker)

would there be benefit of using `varchar(N)` .. where `N` is say `100`? I know there's no performance advantage of varchar over text, but perhaps making it explicit that the size of data that can be stored has an upper limit (text is 'unlimited')

Reply:
Possibly - I don't mind.
I just wanted to make it consistent (type-wise) with `status` https://github.com/alphagov/pay-ledger/blob/32b426629446ca325fa7e6f73c77a1cda7e30114/src/main/resources/migrations/00001_create_table_transaction.sql#L12

Reply:
Updated to `VARCHAR(100)`. Need reapproval.